packages/eslint-config-angular/template.js

```diff
@@ -1,39 +1,30 @@
 module.exports = {
     extends: [
-        'plugin:@angular-eslint/template/recommended'
+        'plugin:@angular-eslint/template/recommended',
+        'plugin:@angular-eslint/template/accessibility'
     ],
     rules: {
         /*
             Overrides to Angular template recommended rules:
-            https://github.com/angular-eslint/angular-eslint/blob/master/packages/eslint-plugin-template/src/configs/recommended.json
+            https://github.com/angular-eslint/angular-eslint/blob/main/packages/eslint-plugin-template/src/configs/recommended.json
         */
 
         /*
-            Overrides to Angular rules outside of the recommended configuration:
+            Overrides to Angular template accessibility rules:
+            https://github.com/angular-eslint/angular-eslint/blob/main/packages/eslint-plugin-template/src/configs/accessibility.json
         */
 
-        /* [accessibility] */
-        '@angular-eslint/template/accessibility-alt-text': 'off',
-
-        /* [accessibility] */
-        '@angular-eslint/template/accessibility-elements-content': 'off',
-
-        /* [accessibility] */
-        '@angular-eslint/template/accessibility-label-for': 'off',
-
-        /* [accessibility] */
-        '@angular-eslint/template/accessibility-label-has-associated-control': 'off',
-
-        /* [accessibility] */
-        '@angular-eslint/template/accessibility-table-scope': 'off',
```

Comment:
Why were all the `[accessibility]` markings removed?..
See: https://github.com/ni/javascript-styleguide?tab=readme-ov-file#accessibility

Reply:
Now that all these rules have been collected into `@angular-eslint/template/accessibility`, I'm following the pattern we used for `@angular-eslint/template/recommended`, i.e. one identifying comment before this group of overrides. I could add the `[accessibility]` tag to that comment, or update the doc so it doesn't reference a tag.

I should note that Angular doesn't classify `no-positive-tabindex` as an accessibility rule, so I removed the tag there, too.

Reply:
Don't think I follow your response from the readme:

```
Evaluate project-specific rule groups
Several sets of rules may be enabled based on requirements of a given project. By default the following sets of rules are in an inert / disabled state, but should be evaluated for your integration.

Text search for the tag associated with a specific rule group in the repository to find the related rules. If enabling a rule group, the rules should be toggled from 'off' to 'error' unless the rule comment says otherwise.
```

And then the section of the README I linked:

```
Tag: [[accessibility]](https://github.com/ni/javascript-styleguide/search?q=%5Baccessibility%5D&type=code)

There currently isn't an NI organization wide requirement to enforce accessibility in applications. The rule group should be enabled if individual applications prioritize accessibility.
```

This PR deleted all the accessibility tags for some reason. So now there is nothing in that rule group for apps to choose to enable. Searching for that tag shows no results:

![image](https://github.com/ni/javascript-styleguide/assets/1588923/29c6912a-ca43-4bce-a0b1-e60a64c037cc)

Reply:
It's fine, I can put all the tags back. I'll also go back to not extending [`@angular-eslint/template/accessibility`](https://github.com/angular-eslint/angular-eslint/blob/main/packages/eslint-plugin-template/src/configs/README.md#accessibility), and instead we'll list each rule explicitly. I do think it makes sense to group these accessibility rules together, though, under a comment that links to the premade accessibility config.

Reply:
> I should note that angular-eslint doesn't classify no-positive-tabindex as an accessibility rule, so I removed the tag there, too.

The tags are about telling apps what accessibility rules exist and dealing with positive tab indexes is definitely accessibility related: https://dequeuniversity.com/tips/tabindex-positive-numbers

Reply:
re-opening comment since looks like there was a race on the last comment

Reply:
It's weird (and kind of annoying) that they didn't include `no-positive-tabindex` in their accessibility preset. It might be a mistake:
![image](https://github.com/ni/javascript-styleguide/assets/7282195/f7763554-6694-403e-a30f-a5475d52cc7e)

I'll add the tag and put it with the other accessibility rules.

Reply:
> I do think it makes sense to group these accessibility rules together, though, under a comment that links to the premade accessibility config.

I'm not necessarily asking for that, just to make it clear how to make accessibility rules discoverable to apps based on the docs. I pointed out your change made it so the README made the rules not easy to discover. Reverting the change is an acceptable alternative. I offered to chat offline to try and understand what you think is clearer. But sounds like you would prefer not to engage in that discussion which is fine too.